docker-compose.yml

```diff
@@ -12,45 +12,45 @@
 version: "3"
 
 services:
-#  wireguard:
-#    image: ghostserverd/wireguard
-#    container_name: wireguard
-#    restart: always
-#    privileged: true
-#    sysctls:
-#      - "net.ipv6.conf.all.disable_ipv6=0"
-#      - "net.ipv6.conf.default.forwarding=1"
-#      - "net.ipv6.conf.all.forwarding=1"
-#      - "net.ipv4.ip_forward=1"
-#    cap_add:
-#      - NET_ADMIN
-#      - SYS_MODULE
-#    networks:
-#      default:
-#        aliases:
-#          - nzbget
-#          - transmission
-#    ports:
-#      - "${NZBGET_PORT}:6789"
-#      - "${TRANS_WEBUI_PORT}:${TRANS_WEBUI_PORT}"
-#      # TRANS_CONNECTION_PORT may need to be port forwarded by your VPN
-#      - "${TRANS_CONNECTION_PORT}:${TRANS_CONNECTION_PORT}"
-#      - "${TRANS_CONNECTION_PORT}:${TRANS_CONNECTION_PORT}/udp"
-#    environment:
-#      - LOCAL_NETWORK=${LOCAL_NETWORK}
-#      # NETWORK_NAME is not needed unless you are mounting docker.sock to write /etc/hosts
-#      # - NETWORK_NAME=${NETWORK_NAME}
-#      # this is required unless you are mounting docker.sock to write /etc/hosts
-#      - IPTABLES_MASQ=0
-#      # space separated  list of docker service names to make addressable
-#      - SERVICE_NAMES=filebot sonarr radarr bazarr jackett announcedd heimdall tautulli plex
-#      # local TLD to use when routing DNS requests which requires aliases for each service to be addressable from within the wireguard network
-#      # - LOCAL_TLD=ghost
-#    volumes:
-#      - ${CONFIG_DIR}/wireguard:/etc/wireguard
-#      - /lib/modules:/lib/modules
-#      # this is no longer necessary but if enabled, will cause the /etc/hosts file to be written with all available services
-#      # - /var/run/docker.sock:/var/run/docker.sock:ro
+  #  wireguard:
```

Comment:
I'm guessing this is getting picked up by linting. I'd actually like to keep it the way it was because with the old way, when the `#` are removed, everything is spaced how it's supposed to be.